docs/source/guide/prompts_overview.md

```diff
@@ -0,0 +1,105 @@
+---
+title: Prompts overview
+short: Prompts overview
+tier: enterprise
+type: guide
+order: 0
+order_enterprise: 225
+meta_title: Prompts overview
+meta_description: An overview of the Prompts feature in the HumanSignal platform
+section: Prompts
+date: 2024-05-15 14:30:14
+---
+
+
+Use Prompts to evaluate and refine your LLM prompts, and then generate predictions to automate your labeling process. 
+
+All you need to get started is an OpenAI API key and a dataset with some ground truth annotations. 
+
+With Prompts, you can:
```

Comment:
we can add a bullet point about leveraging subject matter expertise to rapidly bootstrap projects with labels to get started with ML development faster